server/repository/src/database_settings.rs

```diff
@@ -1,7 +1,6 @@
 use crate::db_diesel::{DBBackendConnection, StorageConnectionManager};
 use diesel::connection::SimpleConnection;
-use diesel::r2d2::{ConnectionManager, ManageConnection, Pool};
-use log::info;
+use diesel::r2d2::{ConnectionManager, Pool};
```

Comment:
just tidying up the warnings - I don't usually run sqlite, but did so this time to test stuff